src/components/Navbar/Navbar.module.scss

```diff
@@ -100,3 +105,20 @@ $default-hamurger-spacing: 10px;
     padding: $ML-spacing 0;
   }
 }
+
+.overlay {
+  z-index: 0;
+  opacity: 0;
+  background-color: $black;
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  transform: scale(0);
```

Comment:
Czy wart robić nam `transform` jeśli nie mamy `transition` na elemencie? Nawet jeśli je usuniemy to dalej będziemy mieli taki sam efekt, co nie? Imo, wygląda to dobrze jak jest teraz, więc chyba możemy usunąć te `transformy` 😉

Reply:
miałoby to sens w odwrotnej kolejności 🙃